src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,117 @@
+package mate.academy.spring.config;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.Movie;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.MovieService;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    private final RoleService roleService;
+    private final UserService userService;
+    private final MovieService movieService;
+    private final CinemaHallService cinemaHallService;
+    private final MovieSessionService movieSessionService;
+
+    public DataInitializer(RoleService roleService,
+                           UserService userService,
+                           MovieService movieService,
+                           CinemaHallService cinemaHallService,
+                           MovieSessionService movieSessionService) {
+        this.roleService = roleService;
+        this.userService = userService;
+        this.movieService = movieService;
+        this.cinemaHallService = cinemaHallService;
+        this.movieSessionService = movieSessionService;
+    }
+
+    @PostConstruct
+    public void inject() {
+        Role adminRole = new Role();
+        adminRole.setRoleName(Role.RoleName.ADMIN);
+        roleService.add(adminRole);
+        Role userRole = new Role();
+        userRole.setRoleName(Role.RoleName.USER);
+        roleService.add(userRole);
+        User admin = new User();
+        admin.setEmail("admin@i.ua");
+        admin.setPassword("admin123");
+        admin.setRoles(Set.of(adminRole));
+        userService.add(admin);
+    }
+
+    @PostConstruct
```

Comment:
Do not have two PostConstruct methods. Create one PostConstruct  init() method and call there one by one two private methods injectMainData and injectAdditionalData